pkg/clusterresourceoverride/mutator.go

```diff
@@ -50,6 +55,40 @@ func (m *podMutator) Mutate(in *corev1.Pod) (out *corev1.Pod, err error) {
 	return
 }
 
+const (
+	SpcType                = "spc_t"
+	SelinuxRelabelResource = "selinuxrelabelfix"
+	SelinuxRelabelGroup    = "admission.node.openshift.io"
+)
+
+var (
+	SelinuxFixEnabledLabelName = fmt.Sprintf("%s.%s/enabled", SelinuxRelabelResource, SelinuxRelabelGroup)
```

Comment:
Do you anticipate `SelinuxFixEnabledLabelName` to change? Otherwise, it could be a constant.

Reply:
I think you should add docs to the exported constants/vars. i.e.
```go
// SelinuxFixEnabledLabelName is the name of the label applied to the pod
const SelinuxFixEnabledLabelName = fmt.Sprintf("%s.%s/enabled", SelinuxRelabelResource, SelinuxRelabelGroup)
```

Reply:
Added the doc. Thanks!